docs/source/data.rst

```diff
@@ -0,0 +1,16 @@
+=========================
+Data
+=========================
+
+
+Access to Disdometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.
```

Comment:
```suggestion
Access to the disdrometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.
```

Reply:
I gave you adming access. Move the repo to the ltelab, rename it disdrodb-data and then modify this accordingly?

Reply:
@regislon ping to do this :)